chia/wallet/wallet.py

```diff
@@ -331,15 +326,8 @@ async def _generate_unsigned_transaction(
         Generates a unsigned transaction in form of List(Puzzle, Solutions)
         Note: this must be called under a wallet state manager lock
         """
-        if primaries_input is None:
-            primaries: Optional[List[AmountWithPuzzlehash]] = None
-            total_amount = amount + fee
-        else:
-            primaries = primaries_input.copy()
-            primaries_amount = 0
-            for prim in primaries:
-                primaries_amount += prim["amount"]
-            total_amount = amount + fee + primaries_amount
+        primaries = [] if primaries_input is None else primaries_input
```

Comment:
This isn't exactly the same as it was before right?
```suggestion
        primaries = None if primaries_input is None else primaries_input
```

Reply:
nvm didn't read far enough :)